### PR TITLE
PXP-503: [CLI] Implement the url-mode flag

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-503](https://mindvalley.atlassian.net/browse/PXP-503)

### Added
- [x] add `url-mode` flag for `wukong application logs` command
screenshot (51)

[PXP-503]: https://mindvalley.atlassian.net/browse/PXP-503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/97#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/feat/application-logs-tonic/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/commands/application/logs.rs
Comment (onimsha) [inline src/commands/application/logs.rs:184]: I think we should format the output a little bit, what do you think of this ?
```suggestion
                    "Copy and paste the 🔗 below to your browser:\n{}?project={}",
```

Comment (jk-gan) [reply src/commands/application/logs.rs:184]: ya sounds good.

Comment (jk-gan) [reply src/commands/application/logs.rs]: updated

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/97#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/97#pullrequestreview-1408383874" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/feat/application-logs-tonic/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
